minimalkv/net/gcstore.py

```diff
@@ -55,16 +55,34 @@ def __init__(
         bucket_creation_location: str = "EUROPE-WEST3",
         project=None,
     ):
+        if isinstance(credentials, str):
+            # Parse JSON from path to extract project name
+            try:
+                with open(credentials) as f:
+                    credentials_dict = json.load(f)
+                    project = project or credentials_dict["project_id"]
+            except (FileNotFoundError, json.JSONDecodeError) as error:
+                print(error)
+                pass
```

Comment:
Why do we print the error and then pass?

Reply:
The pass is pointless 👍🏼